cs/src/Contracts/TunnelOptions.cs

```diff
@@ -12,6 +13,8 @@ namespace Microsoft.DevTunnels.Contracts
     /// </summary>
     public class TunnelOptions
     {
+        private const int HeaderMaxLength = 500;
```

Comment:
This is 255 + 1 + 5, max DNS name length (255) + 1 for ':' + 5 for '65535', max port length.